runtime/vm/FastJNI_java_lang_ref_Reference.cpp

```diff
@@ -51,12 +51,33 @@ Fast_java_lang_ref_Reference_reprocess(J9VMThread *currentThread, j9object_t rec
 	}
 }
 
+/* java.lang.ref.Reference: static private native boolean waitForReferenceProcessingImpl(); */
+jboolean JNICALL
+Fast_java_lang_ref_Reference_waitForReferenceProcessingImpl(J9VMThread *currentThread, j9object_t classObject)
+{
```

Comment:
Can you modify this to be a single return point as well?

Reply:
updated